.github/workflows/validate-docs.yml

```
@@ -0,0 +1,212 @@
+          
+          # Make it executable
+          sudo chmod +x /usr/local/bin/retry_command
+
+      - name: Install dependencies
+        id: install
+        timeout-minutes: 5
+        run: |
+          echo "::group::Install root dependencies"
+          /usr/local/bin/retry_command "bun install --frozen-lockfile"
+          echo "::endgroup::"
+          
```

yamllint (1.35.1)

[error] trailing spaces

(trailing-spaces)